CRM/Contact/BAO/GroupNestingCache.php

```diff
@@ -69,7 +69,7 @@ static public function update() {
     }
 
     if (self::checkCyclicGraph($tree)) {
-      CRM_Core_Error::fatal(ts('We detected a cycle which we cant handle. aborting'));
+      CRM_Core_Error::fatal(ts("We detected a cycle which we can't handle. aborting"));
```

Comment:
Ah, good point. From loooking at the code, it seems like this is an error that can only happen if an admin makes the mistake of importing groups who have a parent group that can also be a child group, creating a cycle. Very unlikely to be a user error. Although that is a rather subjective, and you can expect me to flip-flop on that issue. I guess these days I'm more in a "all messages should be clear and translated" state of mind. However, not a big deal imho for this error message (I googled it, and besides code references, didn't see a forum post about it).